source/inmemory.rst

```diff
@@ -46,28 +46,52 @@ have to do one of the following:
      This is controlled using
      the :option:`--inMemoryStatisticsLogDelaySecs` option.
 
-* If you want to permanently switch to Percona Memory Engine and do not have any
-  valuable data in your database, clean out the default data directory and edit
-  the configuration file:
+* If you want to permanently switch to |inmemory| and do not have any
+  valuable data in your database, do the following:
+
+  Clean out the default data directory:
 
   .. code-block:: bash
 
      $ service mongod stop
      $ rm -rf /var/lib/mongodb/*
-     $ sed -i '/engine: .*inMemory/s/#//g' /etc/mongod.conf
-     $ service mongod start
 
-* If there is data that you want to migrate and make compatible with Percona
-  Memory Engine, use the ``mongodump`` and ``mongorestore`` utilities:
+  Edit the configuration file:
 
   .. code-block:: bash
-
-     $ mongodump --out <dumpDir>
-     $ service mongod stop
-     $ rm -rf /var/lib/mongodb/*
+		 
      $ sed -i '/engine: .*inMemory/s/#//g' /etc/mongod.conf
      $ service mongod start
-     $ mongorestore <dumpDir>
+
+  The :bash:`sed -i '/engine: .*inMemory/s/#//g' /etc/mongod.conf` command uncomments the lines
+  that contain the ``engine: .*inMemory`` pattern in the :file:`/etc/mongod.conf`.
+  
+* If there is data that you want to migrate and make compatible with |inmemory|, use the following methods:
+
+  - for replicasets, use the "rolling restart" process. 
+    Switch to the |inmemory| on the secondary node. Clean out the default data directory and edit the configuration file:
+
+    .. code-block:: bash
+
+       $ service mongod stop
+       $ rm -rf /var/lib/mongodb/*
+       $ sed -i '/engine: .*inMemory/s/#//g' /etc/mongod.conf
```

Comment:
It's not a one-liner, or a fixed number of steps, to change the storage options for any given situation. It's probably involves setting two options only, but how many ones for the other storage engine need to be disabled is non-deterministic matter. So we tell them what to do by description only I think.

$ #Update the mongod.conf file to the new storage.engine value, 
$ # and set the relevant engine-specific settings such as 
$ # storage.wiredTiger.engineConfig.cacheSizeGB or 
$ # storage.inMemory.engineConfig.inMemorySizeGB